### DISCUSSION TITLE
Running winget on remote computers

### DISCUSSION THREAD

**Discussion by rvtdadmin** [microsoft/winget-cli#2414] [Q&A]
Has anyone figured out how to run it on remote computers with Invoke-command, pssession, start-process or any other powershell or non-powershell way?
Without using Task Scheduler or MDM.
I'll use intune for pushing out overall apps but need to be able to run on the fly to fix and clean up apps as we implement winget. 
Thanks

**Comment by AriLobster** [comment]
Hi,

please look forward to following request.

https://github.com/microsoft/winget-cli/issues/2721

At the moment it's not possible.

Kind Regards
Ari

**Comment by MaxKhluptsov** [comment]
I did the following:
1. Configured WinRM on target VMs
2. Now I'm able to run software update by running the following [script](https://pastebin.com/xStwyDEn)

**Reply by rvtdadmin** [reply]
Great. Thanks for putting this here. We'll give it a go.
